core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
I was curious what Stroustroup says, so I just looked in his book, and he says, among other things:

"If you don't like the complications, just initialize consistently."